guides/akashnet-2-software-upgrade.md

```diff
@@ -0,0 +1,49 @@
+# Akashnet-2 Software upgrade procedure
+
+## What validators need to do before the upgrade time?
+
+- Nothing, just sit back and relax. The `software-upgrade` proposal will force the binary (current version, `akash-v0.10.x`) to halt at specified height, i.e., 455200
+
+## What validators need to do at the time of upgrade (i.e., at block height `455200`)
+### 1. Cross check your consensus height:
+```sh
+akash status 2>&1 | jq -r '.SyncInfo.latest_block_height'
```

Comment:
The process won't exit?

Reply:
Since many are using systemctl, it will restart the node again.